### DISCUSSION TITLE
How to configure "Fill in the blanks" test to accept more than one answer or spelling

### DISCUSSION THREAD

**Discussion by javierlabanca** [chamilo/chamilo-lms#5380] [General] [answered]
Hi everybody! I hope you're doing fantastic!

I was wondering if it was possible to accept more than one correct answer in a a "fill in the blanks" test. For example, students may be asked to complete a sentence where the correct answer can either be "I was not" or "I wasn't". Is this possible? Can somebody help me?

Regards,
Javier

**Comment by NicoDucou** [comment, marked as answer]
Hi @javierlabanca 
To set an answer in fil in the blanks you put [MyAsnwer]
To be able to accept 2 different o more answer you just need to seperate them with ||
In your example it would be [I was not||I wasn't]
This will accept both option as correct.

There is an other configuration possible which is with only one |
In this cas it will present a selector like this : 
![image](https://github.com/chamilo/chamilo-lms/assets/6579268/16c38388-fead-4b02-9c3b-4542b74aba89)

**Reply by javierlabanca** [reply]
Hi @NicoDucou 
Thank you so much for the fast reply.
The double pipe || is working fine! Thank you!

Regards,
Javier
